app/views/layouts/application.html.erb

```diff
@@ -98,6 +98,15 @@
     <%= render "subnavigation" if user_signed_in? %>
 
     <main class="govuk-main-wrapper " id="main-content" role="main">
+
+      <div class="govuk-phase-banner feedback-banner">
+        <p class="govuk-phase-banner__content">
+         <span class="govuk-phase-banner__text">
+           This is a new service – your
+           <a class="govuk-link" href="mailto:govwifi-support@digital.cabinet-office.gov.uk?subject=User Feedback Suggestion">feedback</a> will help us to improve it.
```

Comment:
Do we have any config file in which we could keep this mailto link?

Though not necessarily, I can accept the way it's now as well.